Aliucord/src/main/java/com/aliucord/settings/Crashes.java

```diff
@@ -64,14 +66,63 @@ public Context getContext() {
     @SuppressLint("SetTextI18n")
     public void onViewBound(View view) {
         super.onViewBound(view);
-
         setActionBarTitle("Crash Logs");
 
         Context context = requireContext();
-        LinearLayout layout = (LinearLayout) ((NestedScrollView) ((CoordinatorLayout) view).getChildAt(1)).getChildAt(0);
+        int padding = Utils.getDefaultPadding();
+        int p = padding / 2;
+        File folder = new File(Constants.BASE_PATH, "crashlogs");
```

Comment:
Seeing as this path is used multiple times, perhaps it should be added to Constants as CRASHLOG_PATH or something along those lines